crates/driver/src/infra/config/file/mod.rs

```diff
@@ -178,6 +178,11 @@ pub struct ContractsConfig {
 
     /// Override the default address of the WETH contract.
     pub weth: Option<eth::H160>,
+
+    /// Sets the Ethflow contract address. Without this we cannot detect Ethflow
+    /// orders, which leads to such orders not being solved because it appears
+    /// that the user doesn't have enough sell token balance.
+    pub ethflow: Option<eth::H160>,
```

Comment:
Should this be mandatory?

Reply:
Yeah, we can make this mandatory. We just have to make sure the required infra changes are already there before merging the PR to avoid panics.

Reply:
I would like to avoid this configuration change if possible. I think it should be possible if we correctly simulate balances with pre-interactions for orders.